docs/components/query-frontend.md

```diff
@@ -161,6 +161,12 @@ Flags:
                               Log queries that are slower than the specified
                               duration. Set to 0 to disable. Set to < 0 to
                               enable on all queries.
+      --query-frontend.org-id-header=<http-header-name> ...
```

Comment:
@yeya24  - thanks for the feedback.  Regarding docs, I'm considering adding something to explain the new flag in the `slow-query-log` section of query-frontend.md.  And probably some godoc to explain the new function.  Is that the kind of thing you had in mind?

Reply:
You can simply add this description to the flag and I think it would be clear :+1:

Reply:
Not sure I follow.  I think it's already like that.  This description is the one generated from the flag in cmd/thanos/query_frontend.go.

Reply:
Yes, I mean you can add something like `This ord id will be added to the slow-query-log` to the flag description directly and then regenerate the doc

Reply:
Made an update to the flag description.  Hope it's clearer, but open to suggestions for a better way to describe it.